core/src/main/java/org/apache/iceberg/BaseFile.java

```diff
@@ -134,14 +134,14 @@ public PartitionData copy() {
     this.nullValueCounts = nullValueCounts;
     this.lowerBounds = SerializableByteBufferMap.wrap(lowerBounds);
     this.upperBounds = SerializableByteBufferMap.wrap(upperBounds);
-    this.splitOffsets = copy(splitOffsets);
+    this.splitOffsets = splitOffsets == null ? null : splitOffsets.toArray(new Long[0]);
     this.keyMetadata = ByteBuffers.toByteArray(keyMetadata);
   }
 
   /**
    * Copy constructor.
    *
-   * @param toCopy a generic data file to copy.
+   * @param toCopy   a generic data file to copy.
```

Comment:
Could you revert this change? We try to avoid changes that are non-functional and can cause commit conflicts between pull requests and while cherry-picking commits. In addition, keeping these aligned just leads to more changes. I think it's fine to format these aligned in new code, but I don't think it is a good idea to go back and make extra changes just to align.